test/utils/EnumerableSet.test.js

```diff
@@ -11,29 +11,39 @@ describe('EnumerableSet', function () {
     this.set = await EnumerableSetMock.new();
   });
 
+  async function expectMembersMatch (set, members) {
```

Comment:
This function is bordering on too much cleverness (`Promise.all`, the way to build a range). I don't want to block the release so I'm mentioning this as a soft comment. Our tests are too complex, we should do simpler things.

Reply:
I also had some doubts about doing things this way. The original PR had a subtle bug that was only detected by these complex tests however, so I didn't want to get rid of them (nor to spend too much time refactoring the test suite).